lib/cfg_files_utils.pm

```diff
@@ -14,13 +14,37 @@ package cfg_files_utils;
 use Exporter 'import';
 use strict;
 use warnings;
+use Test::Assert ':all';
+use Data::Dumper;
+$Data::Dumper::Sortkeys = 1;
 
 use testapi;
 
 our @EXPORT = qw(
   validate_cfg_file
+  compare_settings
 );
 
+=head2 compare_settings
+
+  compare_settings($value_1, $value_2)
+
+Method validates that the given hash references contain the
+same data. In any other case, the data from both structures 
+are printed, accompanied by an error message.
+
+=cut
+
+sub compare_settings {
+    my ($value_1, $value_2) = @_;
```

Comment:
As it is a new function, let's add named parameters for more than one.

Reply:
I don't understand this comment. There are two parameters. More than one what?

Reply:
when a function take more than one parameter, we agreed to used always named parameters, i.e.: compare_settings(expected => $value1, actual => $value2) because it is more clear from the caller.

Reply:
Ah, ok. Now I understand.

Reply:
But I disagree, in this function it doesn't matter in which order you give the parameters, plus in my use case, both references represent similar input. What you suggest makes total sense in cases that we have specific order of parameters as input.

Reply:
I will edit it just because of the record_info "Expected" and "Got", but in general, I don't think the rule should apply blindly on every function where we pass more than one parameters, only in cases that it actually adds some clarity.

Reply:
> But I disagree, in this function it doesn't matter in which order you pass the parameters, plus in my use case, both references represent similar input. What you suggest makes total sense in cases that we have specific order of parameters as input.

expected and got are different indeed, if the user of your function switches them, she/he will not know what comes from the system and what comes from the expectations.